backend/src/services/interview/interview.service.ts

```diff
@@ -172,6 +179,32 @@ class Interview {
       })),
     };
   }
+
+  public async getOtherByInterviewId({
+    interviewId,
+    count,
+    page,
+  }: InterviewsGetOtherRequestDto): Promise<
+    EntityPagination<InterviewsGetOtherItemResponseDto>
+  > {
+    const interview = await this.getById(interviewId);
+
+    if (!interview) {
+      throw new InterviewsError({
+        message: ExceptionMessage.INTERVIEW_DOES_NOT_EXIST,
+      });
+    }
+
+    const intervieweeUserId = interview.interviewee.id;
+    const ZERO_INDEXED_PAGE = page - 1;
```

Comment:
```suggestion
    const zeroIndexPage = page - 1;
```
constant cannot be computed